lib/views/desktop/send/widgets/DTSelectOrDropAFile.dart

```diff
@@ -21,15 +23,30 @@ class DTSelectOrDropAFile extends StatefulWidget {
 class _DTSelectOrDropAFile extends State<DTSelectOrDropAFile> {
   @override
   Widget build(BuildContext context) {
+    bool flag = false;
     return Container(
       color: widget.dragEntered
           ? Color(0xff3A2655)
           : Theme.of(context).dialogBackgroundColor,
       child: DropTarget(
         onDragDone: (detail) async {
-          await widget.onFileDropped(detail.files.first.readOnlyFile());
+          if (flag == true) return;
```

Comment:
```suggestion
          if (flag) return;
```
But also, it depends what `flag` means which should be renamed.